pkg/innerring/processors/settlement/calls.go

```diff
@@ -72,51 +59,16 @@ func (p *Processor) HandleIncomeCollectionEvent(e event.Event) {
 		return
 	}
 
-	p.incomeContexts[epoch] = incomeCtx
-
 	err = p.pool.Submit(func() {
+		p.log.Info("start basic income collection",
+			zap.Uint64("epoch", epoch))
 		incomeCtx.Collect()
-	})
-	if err != nil {
-		p.log.Warn("could not add handler of basic income collection to queue",
-			zap.Error(err),
-		)
-
-		return
-	}
-}
-
-func (p *Processor) HandleIncomeDistributionEvent(e event.Event) {
-	ev := e.(BasicIncomeDistributeEvent)
-	epoch := ev.Epoch()
-
-	if !p.state.IsAlphabet() {
-		p.log.Info("non alphabet mode, ignore income distribution event")
-
-		return
-	}
-
-	p.log.Info("start basic income distribution",
-		zap.Uint64("epoch", epoch))
-
-	p.contextMu.Lock()
-	defer p.contextMu.Unlock()
-
-	incomeCtx, ok := p.incomeContexts[epoch]
-	delete(p.incomeContexts, epoch)
-
-	if !ok {
-		p.log.Warn("income context distribution does not exists",
+		p.log.Info("start basic income distribution",
```

Comment:
are both logs still necessary?

Reply:
I think they're still useful. Separate stages of the same process.